### ISSUE TITLE
Config

### ISSUE THREAD

**Issue by aaronsace** [fireice-uk/xmr-stak-cpu#111]
When config is null please can you just use the recommended config instead of asking to copy and paste it? 

You've done all that work to detect the config; why not just run it?

**Comment by fireice-uk** [comment]
Problem is that the config is not always optimal - no piece of code can replace human ingenuity ( when it does we will all be screwed :P )

**Comment by aaronsace** [comment]
I think in most cases that this is good enough. Could you add an option so that I can run the recommend settings on any system with MultiPoolMiner? 

I could create a script to copy and paste the config but I thought I'd reach out before I attempt that.

**Comment by ioojimooi** [comment]
when i load the app it says : 
"configurable dev donation level is set to 1.0%"
but i can't find related comments in the default config file, how to tweak this?

**Comment by fireice-uk** [comment]
It is the logical next step (keep in mind that we are still working on getting it working on multi cpu machines) , one of the major problems is that machine-writing to a human-formatted json usually results in general ugliness. We will see how to do it without making the whole cpu config single line.

**Comment by aaronsace** [comment]
That's great, it will greatly improve the usability and performance for everyone.

**Comment by hannesb264** [comment]
@fireice-uk sorry for highjacking this post but i need help and i have no idea how to create a new thread or anything and this is physically the last thing i can try ><. I have been running nicehash with your xmr cpu stak and after getting everything right with the memmory alloc and increasing virtual mem and locking pages etc etc etc i have one last hurdle. I cannot get the cpu threads to get affinity. Have tried editing the configs which didnt work and ive literally tried every forum searching for answers. You see the cpu is running at max capacity but the hash speed is zero. running on very low tech as just started this little hobby. Every now and then though the cpu hashspeed flares up for a second and the block results accepts. But now i saw that if i manually launch your exe as admin it actually affines the cpu thread but cannot connect to the nicehash miner gui. i run the gui as admin as well but that doesnt seem to help. I believe the only thing standing in the way of y cpu being to function properly is the error of "starting single thread, no affinity". Again i am sorry, but do you have any advice for me?

**Comment by fireice-uk** [comment]
@hannesb264 Sounds like you are running the nicehash fork, please ask on their page. If you are running the latest release, it will suggest a config once you start it.

**Comment by hannesb264** [comment]
I'm running the nicehash xmr cpu stak that was edited by you said so in the runtime window, hence why i came to you for support :)

**Comment by fireice-uk** [comment]
That's correct, but I write a lot of software (I'm a freelancer). This is a spare-time project funded by donations. I don't have the capacity to handle the traffic that nicehash has. Please ask for their support with regards to their fork.

**Comment by metlt** [comment]
How can I specify a specific config in the program start line ./xmr-stak-cpu ?
